pkgs/applications/gis/qgis/unwrapped-ltr.nix

```diff
@@ -131,18 +138,24 @@ in mkDerivation rec {
   ] ++ lib.optional (!withWebKit) "-DWITH_QTWEBKIT=OFF"
     ++ lib.optional withGrass "-DGRASS_PREFIX7=${grass}/grass78";
 
-  postFixup = lib.optionalString withGrass ''
+  postFixup = ''
     # grass has to be availble on the command line even though we baked in
     # the path at build time using GRASS_PREFIX
+  '' + lib.optionalString stdenv.isLinux ''
     wrapProgram $out/bin/qgis \
       --prefix PATH : ${lib.makeBinPath [ grass ]}
+  '' + lib.optionalString stdenv.isDarwin ''
+    wrapProgram $out/QGIS.app/Contents/MacOS/QGIS \
+      --prefix PATH : ${lib.makeBinPath [ grass ]}
```

Comment:
Perhaps we can avoid the duplication here by setting a `exe` variable, which is platform dependent, the referencing that in `wrapProgram`?

Reply:
On this point, even without being able to build, @hexagonal-sun do you know if there is a way to disable the automatic wrapping of all the dylibs without removing `makeWrapper` entirely? (I assume that you may have dealt with this in #131891). If not, to actually get this to work I may actually need to remove makeWrapper and just manually do what make-wrapper.sh does, so it only applies to this one executable, rather than to all the files in the .app.